infobserve/common/pool.py

```diff
@@ -29,3 +31,24 @@ def acquire(self):
             (asyncpg.connection.Connection): A Connection instance.
         """
         return self.pool.acquire()
+
+
+class RedisConnectionPool(metaclass=Singleton):
+    redis = None
+
+    async def init_redis_pool(self, redis):
+        """Initialize Redis connection pool.
+        Arguments:
+            redis (dict): A dict with kwargs to initialize the connection pool.
```

Comment:
I think "A dict containing `host` and `port` for redis" is clearer, since it's not really kwargs